CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+
+## Contributing
+
+We welcome all contributions!  Please head to [issues](https://github.com/Sage-Bionetworks/synapsemonitor/issues) to either file any bugs/feature requests or find a task you want to assist with.  Make sure to assign yourself the task if you decide to work on it.
+
+
+## Coding Style
+We use `flake8` and the settings are described in [setup.cfg](setup.cfg).  The code in this package is also automatically formatted by `black` except for the tests.
+
+
+### Fork and clone this repository
+
+See the [Github docs](https://help.github.com/articles/fork-a-repo/) for how to make a copy (a fork) of a repository to your own Github account.
+
+Then, [clone the repository](https://help.github.com/articles/cloning-a-repository/) to your local machine so you can begin making changes.
+
+Add this repository as an [upstream remote](https://help.github.com/en/articles/configuring-a-remote-for-a-fork) on your local git repository so that you are able to fetch the latest commits.
+
+On your local machine make sure you have the latest version of the `develop` branch:
+
+```
+git checkout develop
+git pull upstream develop
+```
+
+### Install development dependencies
+This will install all the dependencies of the package including the active branch of `synapsemonitor`.
+
+```
+pipenv install --dev
```

Comment:
I actually didn't have pipenv installed by default on my machine.  If people didn't install python via anaconda, they might be lacking black and pytest as well.  Might be nice to add a line at the top with all the tool dependencies for contributing to ensure they have everything installed.